Sources/Core/API/TextureManager.swift

```diff
@@ -24,23 +24,18 @@ public final class TextureManager {
 
     // MARK: - Public
 
-    public func preloadTexture(named name: String, scale: Int? = nil, onQueue queue: DispatchQueue = .main) {
+    public func preloadTexture(named name: String, scale: Int? = nil, textureFormat: TextureFormat = .png, onQueue queue: DispatchQueue = .main) {
```

Comment:
I think here we can also simply refer to the format as `format`, since the context already includes "Texture".